test/com/google/javascript/jscomp/RewriteDynamicImportsTest.java

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright 2021 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.javascript.jscomp;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.javascript.jscomp.RewriteDynamicImports.UNABLE_TO_COMPUTE_RELATIVE_PATH;
+
+import com.google.common.collect.ImmutableList;
+import com.google.javascript.jscomp.CompilerOptions.LanguageMode;
+import com.google.javascript.jscomp.testing.TestExternsBuilder;
+import com.google.javascript.jscomp.type.ReverseAbstractInterpreter;
+import com.google.javascript.jscomp.type.SemanticReverseAbstractInterpreter;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Unit tests for {@link RewriteDynamicImports} */
+@RunWith(JUnit4.class)
+public class RewriteDynamicImportsTest extends CompilerTestCase {
+  private String dynamicImportAlias = "imprt_";
+
+  @Override
+  @Before
+  public void setUp() throws Exception {
+    super.setUp();
+    enableCreateModuleMap();
+    enableTypeInfoValidation();
+    disableScriptFeatureValidation();
+  }
+
+  @Override
+  protected CompilerOptions getOptions() {
+    CompilerOptions options = super.getOptions();
+    options.setLanguage(LanguageMode.ECMASCRIPT_2020);
```

Comment:
Since we're rewriting the dynamic imports, should at least some of these test cases use a lower language out than this?

Reply:
In general, I think that unit tests should be able to remove the language mode completely, since they (recently changed) now default to the most recent language mode with no transpilation.

Reply:
I can test that an error is thrown with a lower language level without an alias. But that's the only impact a lower language level has.

Reply:
I think that Bradford was just trying to forward my comment here, which is that I've been trying to remove the language mode settings from the unit tests completely.  Do the unit tests in this file still work with this line removed?

Reply:
They do - but I added a warning (and test) for when the output language level doesn't support dynamic imports and an alias wasn't specified.